src/src/tx_main.cpp

```diff
@@ -170,7 +170,28 @@ bool ICACHE_RAM_ATTR IsArmed()
 // Assume this function is called inside loop(). Heavy functions goes here.
 void DynamicPower_Update()
 {
+ 
+  // if telemetry is not arrived, quick return.
+  if (!dynamic_power_updated)
```

Comment:
You can't move this up to the top because then DP can't change power without telemetry (e.g. if going to disconnected / armed  etc).

Reply:
Oops that’s true.. I’ll revise